operator-common/src/main/java/io/strimzi/operator/common/operator/resource/StatusUtils.java

```diff
@@ -48,6 +50,25 @@ public static String iso8601(Instant instant) {
         return ZonedDateTime.ofInstant(instant, ZoneOffset.UTC).format(DateTimeFormatter.ISO_INSTANT);
     }
 
+    /**
+     *  Returns an Instant from a string date in ISO 8601 format
+     * @param date a string representing a date, for example "2019-07-23T09:08:12.356Z"
+     * @return an Instant
+     */
+    public static Instant iso8601(String date)  {
```

Comment:
8601 also defined formats for pure dates, pure times and date times without timezones, so the name `iso8601` is a bit ambiguous. I wonder if `isoUtcDatetime` would be better.